core/services/ocr2/delegate.go

```diff
@@ -496,7 +496,7 @@ func GetEVMEffectiveTransmitterID(jb *job.Job, chain legacyevm.Chain, lggr logge
 		if chain == nil {
 			return "", fmt.Errorf("job forwarding requires non-nil chain")
 		}
-		effectiveTransmitterID, err := chain.TxManager().GetForwarderForEOA(common.HexToAddress(spec.TransmitterID.String))
+		effectiveTransmitterID, err := chain.TxManager().GetForwarderForEOAOCR2(common.HexToAddress(spec.TransmitterID.String), common.HexToAddress(spec.ContractID))
```

Comment:
is the existing GetForwarderForEOA method still used somewhere else?

Reply:
I won't touch keepers for now as I am not 100% sure if it makes sense there and eth tx task shouldn't use this and will get deprecated anyway

Reply:
Also trying to minimize the scope as this affects the migration